src/Components/Web.JS/src/Platform/WebAssemblyStartOptions.ts

```diff
@@ -1,3 +1,5 @@
+import { BootConfigResult, BootJsonData } from "./BootConfig";
```

Comment:
I wonder if there's a .editorconfig for this

```suggestion
import { BootConfigResult, BootJsonData } from './BootConfig';
```

Reply:
We had TSLint enabled at some point, but I think we end up ignoring it.